main.go

```diff
@@ -34,10 +34,15 @@ type config struct {
 
 	// Omited fields
 	EnvOmittedFields []string `envconfig:"ENV_OMITTED_FIELDS"`
+
+	// Site env
+	DadosJusUrl    string `envconfig:"DADOSJUS_URL" required:"true"`
```

Comment:
```suggestion
	DadosJusURL    string `envconfig:"DADOSJUS_URL" required:"true"`
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms